### PR TITLE
[test] Update perf test to use npx over yarn

### PR DESCRIPTION
**Pull request by smyrick**
## :pencil: Description
Drop the need for developers to have yarn installed and instead use npx which is included with npm

### CODE REVIEW

File: .github/workflows/pr-check-docs.yml [hunk 1]
```diff
@@ -21,9 +22,27 @@ jobs:
...
 
-    - name: Run npm commands
+    - name: Build docs
       if: github.repository == 'ExpediaGroup/graphql-kotlin'
       run: |
         cd website
         npm install
         npm run build
+
+  check-perf-tests:
```
Comment (dariuszkuc) [inline .github/workflows/pr-check-docs.yml]: why would we want to run perf tests if someone updates docs?

Comment (dariuszkuc) [reply .github/workflows/pr-check-docs.yml]: maybe we should split those up?

Comment (smyrick) [reply .github/workflows/pr-check-docs.yml]: We may just be able to delete the perf tests here that use npm. Instead we may want some other client. This was an add in to debug our subscriptions.

**Comment by smyrick** [comment]
Closing in favor of https://github.com/ExpediaGroup/graphql-kotlin/pull/576

**Pull request closed without merging**